algo/heap_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build unit_test
```

Comment:
you don't need a tag for unit test. No tag means it is a unit test

Reply:
done